diff -Nur xbill-2.0.orig/Cable.h xbill-2.0/Cable.h
--- xbill-2.0.orig/Cable.h	Thu Jan 23 17:22:49 1997
+++ xbill-2.0/Cable.h	Fri Jan  5 20:02:21 2001
@@ -13,7 +13,7 @@
 	void setup();
 	void draw();
 	void update();
-	onspark (int locx, int locy);
+	int onspark (int locx, int locy);
 
 };
 
--- xbill-2.0/UI.cc.orig	1997-01-23 17:22:51.000000000 +0100
+++ xbill-2.0/UI.cc	2006-10-16 23:07:56.461712750 +0200
@@ -125,10 +125,10 @@
 void UI::popup_dialog (int dialog) {
 	Widget w;
 	switch (dialog) {
-		case game.ENTERNAME: w = enternamebox; break;
-		case game.HIGHSCORE: w = highscorebox; break;
-		case game.SCORE: w = scorebox; break;
-		case game.ENDGAME: w = endgamebox; break;
+		case Game::ENTERNAME: w = enternamebox; break;
+		case Game::HIGHSCORE: w = highscorebox; break;
+		case Game::SCORE: w = scorebox; break;
+		case Game::ENDGAME: w = endgamebox; break;
 	}
 	popup (NULL, &w, NULL);
 }
@@ -139,11 +139,11 @@
 
 void UI::set_cursor(int cursor) {
 	switch (cursor) {
-	case game.BUCKETC:
+	case Game::BUCKETC:
 		XDefineCursor(display, window, bucket.cursor.cursor); break;
-	case game.DOWNC:
+	case Game::DOWNC:
 		XDefineCursor(display, window, downcursor.cursor); break;
-	case game.DEFAULTC:
+	case Game::DEFAULTC:
 		XDefineCursor(display, window, defaultcursor.cursor); break;
 	default:
 		XDefineCursor(display, window, OS.cursor[cursor].cursor);
